pkg/v4/resource/certconfig/create.go

```diff
@@ -23,7 +22,7 @@ func (r *Resource) ApplyCreateChange(ctx context.Context, obj, createChange inte
 		r.logger.LogCtx(ctx, "level", "debug", "message", "creating certconfigs")
 
 		for _, certConfigToCreate := range certConfigsToCreate {
-			_, err = r.g8sClient.CoreV1alpha1().CertConfigs(v1.NamespaceDefault).Create(certConfigToCreate)
+			_, err = r.g8sClient.CoreV1alpha1().CertConfigs(certConfigToCreate.ObjectMeta.Namespace).Create(certConfigToCreate)
```

Comment:
Nitpick: you don't have to call `.OjbectMeta` explicitly.

Reply:
True. I tend to forget this, but then again, I do prefer the explicit form. When reading the code, seeing that shortening would not necessarily be obvious and therefore it would make reading more difficult.

Reply:
Oh, I misunderstood. I had something else in my mind actually. Sure. This is a good shout. Thanks!